supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -68,8 +70,10 @@ Use the following template text verbatim, where _<feature_name>_ is your feature
 ----
 [IMPORTANT]
 ====
-_<feature_name>_ is a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-
+_<feature_name>_ is a Technology Preview feature only.
+Technology Preview software is not intended for production or business-critical workloads.
```

Comment:
I don't think we previously referred to it as "Technology Preview _software_". Just "Technology Preview _features_". Is that a purposeful change? If not, could we change it to say "Technology Preview features..."?